source/locales/zh-CN.json

```diff
@@ -14,8 +14,8 @@
   "Email Alternatives": "电子邮件的替代品",
   "Email Clients": "电子邮件客户端",
   "Email Encryption": "电子邮件加密",
-  "Enterprise Suite": "Enterprise Suite",
-  "enterprise-suite-notes": "The enterprise suite category is for solutions for organizations that cover more than 10 categories in an integrated fashion (ex.: logins work throughout all apps, etc.)",
+  "Enterprise Suite": "企业套件",
+  "enterprise-suite-notes": "“企业套件”分类中列出了一些适合公司或组织使用的软件套装，往往是包含了10种以上不同种类工具的合集（大都支持统一登录等特性）。",
```

Comment:
guys, sorry for my disappear these months. i have some bug to fix in my real-life :( .

`“”` is quote in Chinese, used in pair. You can use it to quote the word you are defining, just like the bold word at the beginning of every Wikipedia entry.